tasks/webpack.js

```diff
@@ -64,7 +66,13 @@ module.exports = (grunt) => {
           }
         }
 
-        if (!opts.keepalive) done();
+        if (!opts.keepalive) {
```

Comment:
This will have weird  effects if different targets have different `keepalive` set. Can we instead maybe check every target and if at least one target has keepalive===true then do it?
So basically  outside the forEach have
```
let keepalive = false;
```
and then for every target:
```
keepalive = keepalive || opts.keepalive;
```

Als the counter decrease has to be outside this if, as otherwise we can potentially never reach 0 right now if on target has keepalive true.

so maybe just simplify to
```
if (--runningTargetCount === 0 && !keepalive) done();
```